src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -246,7 +255,7 @@ async def create_privacy_request(
                 queue_privacy_request(privacy_request.id)
         except EmailDispatchException as exc:
             kwargs["privacy_request_id"] = privacy_request.id
-            logger.error("EmailDispatchException: %s", exc)
+            logger.error("EmailDispatchException: %s", Pii(exc))
```

Comment:
This will mask the entirety of all EmailDispatchExceptions, and most of your EmailDispatchExceptions are very helpful without revealing PII.

I'd instead just wrap this specific exception in the email dispatch service : `raise EmailDispatchException(f"Email failed to send due to: {Pii(e)}")`

Reply:
oh good idea!